command/repo.go

```diff
@@ -75,15 +76,17 @@ To pass 'git clone' flags, separate them with '--'.`,
 var repoCreateCmd = &cobra.Command{
 	Use:   "create [<name>]",
 	Short: "Create a new repository",
-	Long:  `Create a new GitHub repository`,
-	Example: utils.Bold("$ gh repo create") + `
```

Comment:
We should not use ANSI escape sequences for terminal colors here because this text is also used to generate man pages and HTML pages, where those ANSI sequences are not supported.